Create seed phrase option on add-key

tituszban opened the pull request:
The problem:
There is currently no way in the CLI to add a seed phrase to an already existing account. As a result, if an account was generated on the CLI, it is not easy to also add it on the web wallet.

The solution:
If `add-key` is called not with a public key, but with the `--seedPhrase` option, the key generated by that seed phrase is added to the account

MattKetmo commented:
yes my bad, I confused add-key & generate-key when talking about "overwrites".

A few things about your code:

- Setting `access-key` `required` to `false` changes the behavior of the command. 
  It's no more a 2nd arg but an option `--access-key`. Which means part 1 in README should be edited too (as well as doc pages)

- Also, since it's a non-required option, a non-empty check should be added before calling `await account.addKey(options.accessKey` at line 55 in order to avoid this error

```
Adding full access key = undefined to example.testnet.
An error occured
TypeError: Cannot read properties of undefined (reading 'borshSerialize')
    at serializeStruct (/private/tmp/near-cli/node_modules/borsh/lib/index.js:302:20)
...
```

- I would not `console.log` the mnemonic for security reasons (or maybe you wanted to write `seedPhrasePublicKey` at line 44?)

Otherwise yes I think this improve ux. We could even have an option to auto generate the seed phrase.

tituszban commented:
* Updated the readme, and PR to update the docs: https://github.com/near/docs/pull/1102
* Added a non-empty check
* Removed seed phrase from console log

MattKetmo commented on commands/add-key.js:
```diff
@@ -5,13 +5,13 @@ const { utils } = require('near-api-js');
 
 module.exports = {
-    command: 'add-key <account-id> <access-key>',
+    command: 'add-key <account-id>',
```
I noticed there is another syntax to possibly use the existing arg as an optional parameter
```suggestion
    command: 'add-key <account-id> [access-key]',
```

example in the [generate-key command](https://github.com/near/near-cli/blob/4b4a787d0eafaa97fb7c2c741cdfaacf2c53f35b/commands/generate-key.js#L6)

This would allow to not break the current input of the command (and thus the existing docs). But I let maintainers decide for the best solution :)

tituszban replied:
I think that's a pretty good idea. Other thoughs?
